target/snitch_cluster/Makefile

```diff
@@ -47,6 +47,7 @@ CLUSTER_GEN_SRC ?= $(wildcard $(ROOT)/util/clustergen/*.py)
 
 BIN_DIR           ?= bin
 GENERATED_DIR     ?= $(MKFILE_DIR)generated
+PERIPH_DIR        ?= $(ROOT)/hw/snitch_cluster/src/snitch_cluster_peripheral
```

Comment:
I think we can stop tracking the generated `snitch_cluster_peripherals` files and move them to `GENERATED_DIR` as well.

Reply:
Yeah, why not. I will change it

Reply:
I moved them to the `generated` directory now